stories/navbar.stories.js

```diff
@@ -0,0 +1,180 @@
+      <Section.Item>
+        Home
+      </Section.Item>
+      <Section.Item>
+        About
+      </Section.Item>
+      <Section.Item>
+        Contact
+      </Section.Item>
+    </Nav.Center>
+    <Nav.Right as |Section|>
+      <Section.Icon
+        @icon="dashboard"
+      />
+      <Section.Icon
+        @icon="menu-dialpad"
+      />
+      <Section.Icon
+        @icon="user-profile-circle"
+      />
+    </Nav.Right>
+  </DenaliNavbar>
+  `,
+});
+
+export const Items = () => ({
+  template: hbs`
+  <DenaliNavbar
+        @isResponsive={{isResponsive}}
+        @isMenuActive={{isMenuActive}}
+        as |Nav| >
+    <Nav.Left as |Section|>
+      <Section.Logo src="https://denali-design.github.io/denali-css/denali-logo.svg"/>
+    </Nav.Left>
+    <Nav.Right as |Section|>
+      <Section.Item @isActive={{true}}>
+        Home
+      </Section.Item>
+      <Section.Item>
+        About
+      </Section.Item>
+      <Section.Item>
+        Contact
+      </Section.Item>
+    </Nav.Right>
+  </DenaliNavbar>
+  `,
+});
+
+export const Icons = () => ({
+  template: hbs`
+  <DenaliNavbar
+        @isResponsive={{isResponsive}}
+        @isMenuActive={{isMenuActive}}
+        as |Nav| >
+    <Nav.Left as |Section|>
+      <Section.Logo src="https://denali-design.github.io/denali-css/denali-logo.svg"/>
+    </Nav.Left>
+    <Nav.Right as |Section|>
+      <Section.Icon
+        @icon="dashboard"
+      />
+      <Section.Icon
+        @icon="menu-dialpad"
+      />
+      <Section.Icon
+        @icon="user-profile-circle"
+      />
+    </Nav.Right>
+  </DenaliNavbar>
+  `,
+});
+
+export const Left = () => ({
+  template: hbs`
+  <DenaliNavbar
+        @isResponsive={{isResponsive}}
+        @isMenuActive={{isMenuActive}}
+        as |Nav| >
+    <Nav.Left as |Section|>
+      <Section.Logo src="https://denali-design.github.io/denali-css/denali-logo.svg"/>
+    </Nav.Left>
+  </DenaliNavbar>
+  `,
+});
+
+export const Center = () => ({
+  template: hbs`
+  <DenaliNavbar
+        @isResponsive={{isResponsive}}
+        @isMenuActive={{isMenuActive}}
+        as |Nav| >
+    <Nav.Left as |Section|>
+      <Section.Logo src="https://denali-design.github.io/denali-css/denali-logo.svg"/>
+    </Nav.Left>
+    <Nav.Center as |Section|>
+      <Section.Item>
+        Center
+      </Section.Item>
+    </Nav.Center>
+  </DenaliNavbar>
+  `,
+});
+
+export const Right = () => ({
+  template: hbs`
+  <DenaliNavbar
+        @isResponsive={{isResponsive}}
+        @isMenuActive={{isMenuActive}}
+        as |Nav| >
+    <Nav.Left as |Section|>
+      <Section.Logo src="https://denali-design.github.io/denali-css/denali-logo.svg"/>
+    </Nav.Left>
+    <Nav.Right as |Section|>
+      <Section.Icon
+        @icon="user-profile-circle"
+      />
+    </Nav.Right>
+  </DenaliNavbar>
+  `,
+});
+
+export const Playground = () => ({
+  template: hbs`
+  <DenaliNavbar
+        @isResponsive={{isResponsive}}
+        @isMenuActive={{isMenuActive}}
```

Comment:
Does this do anything at the moment?

Reply:
I added them because it was an option we supported.  I am not sure what this does though.  I can remove these.